docs/tutorials/pytorch2onnx.md

```diff
@@ -23,12 +23,9 @@
 
 ### Prerequisite
 
-1. Please refer to [get_started.md](../get_started.md) for installation of MMCV and MMDetection.
-2. Install onnx and onnxruntime
-
-  ```shell
-  pip install onnx onnxruntime
-  ```
+1. Install the prerequisites following steps 1-3 in [get_started.md/Installation](../get_started.md). (We do not install MMCV and MMDetection because we will do it manually).
+2. Build custom operators for ONNX Runtime and install MMCV following [Custom operators for ONNX Runtime in MMCV](https://github.com/open-mmlab/mmcv/blob/master/docs/onnxruntime_op.md/)
```

Comment:
May use a more accurate reference https://github.com/open-mmlab/mmcv/blob/master/docs/onnxruntime_op.md/#how-to-build-custom-operators-for-onnx-runtime